tests/storage/hostdisk.go

```diff
@@ -26,7 +26,8 @@ import (
 	k8sv1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/util/rand"
 
-	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
+	"kubevirt.io/kubevirt/pkg/virt-config/featuregate"
+
```

Comment:
nit: remove newline

Reply:
The newline seems correct, separating production from e2e tests.